modules/nodes-cluster-resource-override-deploy-cli.adoc

```diff
@@ -130,9 +134,9 @@ spec:
       limitCPUToMemoryPercent: 200 <4>
 ----
 <1> The name must be `cluster`.
-<2> Optional. Specify the percentage to override the container memory limit, if used, between 1-100. The default is 50.
-<3> Optional. Specify the percentage to override the container CPU limit, if used, between 1-100. The default is 25.
-<4> Optional. Specify the percentage to override the container memory limit, if used. Scaling 1Gi of RAM at 100 percent is equal to 1 CPU core. This is processed prior to overriding the CPU request, if configured. The default is 200.
+<2> Optional: Specify the percentage to override the container memory limit, if used, between 1-100. The default is `50`.
+<3> Optional: Specify the percentage to override the container CPU limit, if used, between 1-100. The default is `25`.
+<4> Optional: Specify the percentage to override the container memory limit, if used. Scaling 1Gi of RAM at 100 percent is equal to 1 CPU core. This is processed before overriding the CPU request, if configured. The default is `200`.
```

Comment:
Bit of a nit, but should a space exist between "1" and "Gi"?

![Screenshot from 2024-09-20 09-37-48](https://github.com/user-attachments/assets/a45e8b2b-611b-440c-bf0d-417d04bde0c1)